src/radical/pilot/agent/scheduler/continuous_reconfig.py

```diff
@@ -0,0 +1,57 @@
+
+__copyright__ = 'Copyright 2023, The RADICAL-Cybertools Team'
+__license__   = 'MIT'
+
+import os
+
+import radical.utils as ru
+
+from ... import constants as rpc
+
+from .continuous import Continuous
+
+
+# ------------------------------------------------------------------------------
+#
+class ContinuousReconfig(Continuous):
+
+    # --------------------------------------------------------------------------
+    #
+    def __init__(self, cfg, session):
+
+        super().__init__(cfg, session)
+
+        self._task_reqs = {}  # resource requirements (ranks, cores_per_rank)
+
+    # --------------------------------------------------------------------------
+    #
+    def _set_task_reqs(self):
+
+        reqs_file = ''
+        if os.path.isfile(reqs_file):
+            self._task_reqs.update(ru.read_json(reqs_file))
+
+    # --------------------------------------------------------------------------
+    #
+    def work(self, tasks):
+
+        if not self._task_reqs:
+            self._set_task_reqs()
+
+        if self._task_reqs:
```

Comment:
what happens if that is false - tasks just disappear in that case?

Reply:
reworked